src/common/query.rs

```diff
@@ -147,6 +147,14 @@ pub type MultiQueryBuilder<T> = MultiRpcBuilder<T, RpcQueryRequest, BlockReferen
 pub type ValidatorQueryBuilder<T> = RpcBuilder<T, RpcValidatorRequest, EpochReference>;
 pub type BlockQueryBuilder<T> = RpcBuilder<T, RpcBlockRequest, BlockReference>;
 
```

Comment:
i don't know, this module could be split into some smaller sub-modules, maybe defined by some core trait that all items within a sub-module implement, maybe by some other categories (maybe by a few layers of categories) , 
but the names are all a bit lookalike , QueryBuilder, RpcBuilder, RpcHandler, RpcRequest, RpcRequestHandler, ...


idk, if e.g. `RpcBuilder` resided in `near_api::common::query::<module_name>`, and  `QueryBuilder` resided in 
`near_api::common::query::<module_name>::aliases`, maybe these would be easier to separate from `MultiRpcBuilder` and its alias

it's not large though, only 800 lines

Reply:
Yeah, agree. Some polishing might be done.

Reply:
https://github.com/near/near-api-rs/issues/50